### PR TITLE
add diamond donors to donor slider

### PR DESCRIPTION
**Pull request by tanyaparker**
Now the donor slider shows all diamond donors first, platinum donors second, and gold donors third. Each donor category is shuffled.

**Before**
screen shot 2018-06-29 at 11 13 39 am

**After**
screen shot 2018-06-29 at 11 12 47 am

### CODE REVIEW

File: pegasus/sites.v3/code.org/views/donor_slider.haml [hunk 1]
```diff
@@ -1,23 +1,35 @@
...
           = I18n.t(:homepage_donors_donate)
 
-    .donor-slider{style: "color:white; text-align:center; font-size:13px"}
-      - supporters.each_with_index do |supporter, index|
-        %a{:href=>supporter[:url_s], :target=> "_blank", :style=>"color:white; font-family: 'Gotham 4r', sans-serif;"}
-          - if index == supporters.length-1
+    .donor-slider
+      - diamond_supporters.each_with_index do |supporter, index|
+        %a{:href=>supporter[:url_s], :target=> "_blank"}
+          != "#{supporter[:name_s]}, "
```
Comment (breville) [inline pegasus/sites.v3/code.org/views/donor_slider.haml]: Do we need `!=` here?

Comment (tanyaparker) [reply pegasus/sites.v3/code.org/views/donor_slider.haml]: Hmm. There's no HTML in the name, so I guess not! My bad for copying existing code.

File: pegasus/sites.v3/code.org/views/donor_slider.haml [hunk 2]
```diff
@@ -1,23 +1,35 @@
...
-        %a{href: "/donate", style: "color:white"}
+        %a{href: "/donate"}
           = I18n.t(:homepage_donors_donate)
 
-    .donor-slider{style: "color:white; text-align:center; font-size:13px"}
-      - supporters.each_with_index do |supporter, index|
-        %a{:href=>supporter[:url_s], :target=> "_blank", :style=>"color:white; font-family: 'Gotham 4r', sans-serif;"}
-          - if index == supporters.length-1
+    .donor-slider
+      - diamond_supporters.each_with_index do |supporter, index|
```
Comment (breville) [inline pegasus/sites.v3/code.org/views/donor_slider.haml]: Oh, looks like we don't need `index` for the first two of these iterators.

Comment (tanyaparker) [reply pegasus/sites.v3/code.org/views/donor_slider.haml]: Oops. Shouldn't the robocop thingamajig have caught that? Or does it only scan pure ruby files?

Comment (breville) [reply pegasus/sites.v3/code.org/views/donor_slider.haml]: I believe only Ruby files, which is another reason why it's good to extract Ruby from HAML where we can.

Comment (tanyaparker) [reply pegasus/sites.v3/code.org/views/donor_slider.haml]: I'm more familiar with cleaning up css. Where would I put the ruby code for pegasus files?

**Pull request merged**